core/src/jsMain/kotlin/dev/fritz2/dom/mount.kt

```diff
@@ -221,26 +155,42 @@ fun <V> Flow<List<V>>.eachIndex(): Flow<Patch<V>> =
  * @param upstream the [Flow] that should be mounted
  * @param createPatches lambda defining, how to compare two versions of a [List]
  */
-inline fun <V> TagContext.mountPatches(
-    into: RenderContext?,
+internal fun <V> RenderContext.mountPatches(
+    into: Tag<HTMLElement>?,
     upstream: Flow<List<V>>,
-    crossinline createPatches: (Flow<List<V>>, MutableMap<Node, Job>) -> Flow<List<Patch<RenderContext>>>,
+    createPatches: (Flow<List<V>>, MutableMap<Node, MountPointImpl>) -> Flow<List<Patch<Tag<HTMLElement>>>>,
 ) {
-    val target = if (into != null) {
-        into.domNode.clear()
-        ProxyContext(job, into)
-    } else mountContext(job)
-    val jobs = mutableMapOf<Node, Job>()
+    val target = into?.apply {
+        this.domNode.clear()
+        SET_MOUNT_POINT_DATA_ATTRIBUTE()
+    }
+        ?: div(MOUNT_POINT_STYLE_CLASS, content = SET_MOUNT_POINT_DATA_ATTRIBUTE)
+
+    val mountPoints = mutableMapOf<Node, MountPointImpl>()
 
-    mountSimple(target.job, createPatches(upstream, jobs)) { patches ->
+    mountSimple(target.job, createPatches(upstream, mountPoints)) { patches ->
         patches.forEach { patch ->
             when (patch) {
-                is Patch.Insert -> target.domNode.insert(patch.element, patch.index)
-                is Patch.InsertMany -> target.domNode.insertMany(patch.elements, patch.index)
-                is Patch.Delete -> target.domNode.delete(patch.start, patch.count) { node ->
-                    val job = jobs.remove(node)
-                    if (job != null) job.cancelChildren()
-                    else console.error("could not cancel renderEach-jobs!")
+                is Patch.Insert -> {
+                    target.domNode.insert(patch.element, patch.index)
+                    val mountPointImpl = mountPoints[patch.element.domNode]
+                    if (mountPointImpl != null) mountPointImpl.runAfterMounts()
+                    else console.error("could not run afterMount on inserting ${patch.element}")
```

Comment:
What would be missing is the mount-point. Since all renders (even the global one) are MPs and put themselves on the scope, this would mean that something regarding the scopes does not work. I don't think, that this could be fixed in user code.